src/components/VideoControl.tsx

```diff
@@ -100,8 +100,15 @@ export const VideoControl: FC<VideoControlProps> = ({
     };
   }, [stream, video]);
 
+  const styles = {
+    border: isVideoEnabled ? '1px solid lightgray' : 'none',
+    maWidth: '100%',
+    textAlign: 'center',
+    padding: 10,
+  } as React.CSSProperties;
```

Comment:
Static CSS properties should go in CSS classes. In this case I think you can add them to `video-wrapper`